tensorflow_addons/custom_ops/image/python/distort_image_ops_test.py

```diff
@@ -0,0 +1,347 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may noa use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for python distort_image_ops."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import time
+
+import numpy as np
+
+import tensorflow as tf
+
+from six.moves import xrange
+from tensorflow.python.framework import test_util as tf_test_util
+from tensorflow_addons.custom_ops.image.python import distort_image_ops
+
+# TODO(huangyp): also measure the differences between AdjustHsvInYiq and
+# AdjustHsv in core.
+
+
+class AdjustHueInYiqTest(tf.test.TestCase):
+    def _adjust_hue_in_yiq_np(self, x_np, delta_h):
+        """Rotate hue in YIQ space.
+
+        Mathematically we first convert rgb color to yiq space, rotate the hue
+        degrees, and then convert back to rgb.
+
+        Args:
+          x_np: input x with last dimension = 3.
+          delta_h: degree of hue rotation, in radians.
+
+        Returns:
+          Adjusted y with the same shape as x_np.
+        """
+        self.assertEqual(x_np.shape[-1], 3)
+        x_v = x_np.reshape([-1, 3])
+        y_v = np.ndarray(x_v.shape, dtype=x_v.dtype)
+        u = np.cos(delta_h)
+        w = np.sin(delta_h)
+        # Projection matrix from RGB to YIQ. Numbers from wikipedia
+        # https://en.wikipedia.org/wiki/YIQ
+        tyiq = np.array([[0.299, 0.587, 0.114], [0.596, -0.274, -0.322],
+                         [0.211, -0.523, 0.312]])
+        y_v = np.dot(x_v, tyiq.T)
+        # Hue rotation matrix in YIQ space.
+        hue_rotation = np.array([[1.0, 0.0, 0.0], [0.0, u, -w], [0.0, w, u]])
+        y_v = np.dot(y_v, hue_rotation.T)
+        # Projecting back to RGB space.
+        y_v = np.dot(y_v, np.linalg.inv(tyiq).T)
+        return y_v.reshape(x_np.shape)
+
+    def _adjust_hue_in_yiq_tf(self, x_np, delta_h):
+        with self.cached_session(use_gpu=True):
```

Comment:
can you remove all `session` below?

Reply:
I'm not sure how to set config in eager mode, so I keep the sessions in benchmark classes the same.

Reply:
Would you mind leaving a TODO comment?